workflow/schemas/workflow.schema.json

```
@@ -76,7 +84,7 @@
         "Config": {
             "type": "object",
             "propertyNames": {
-                "$comment": "TODO: check this regex. This is the name of a device class, e.g. 'GilsonPipetMax'. It may be the case that an enum is preferable.",
+                "$comment": "This is the name of a device class, e.g. 'GilsonPipetMax'.",
```

Heh, I wonder: this is essentially an explicit enum in the code, reflecting exactly which devices are supported. I wonder if that should be reflected here in order to avoid accidental typos which would otherwise end up being silently ignored.

---

Ah, got it. Will change.